docs/userguides/transactions.md

```diff
@@ -121,3 +121,63 @@ In your `ape-config.yaml` file, add the following:
 ```yaml
 transaction_acceptance_timeout: 600  # 5 minutes
 ```
+
+## Traces
+
+If you are using a provider that is able to fetch transaction traces, such as the [ape-hardhat](https://github.com/ApeWorX/ape-hardhat) provider, you can call the [`ReceiptAPI.show_trace()`](../methoddocs/api.html?highlight=receiptapi#ape.api.transactions.ReceiptAPI.show_trace) method.
+
+```python
+from ape import accounts, project
+
+owner = accounts.load("acct")
+contract = project.Contract.deploy(sender=owner)
+receipt = contract.methodWithoutArguments()
+receipt.show_trace()
+```
+
+The trace might look something like:
+
+```bash
+Call trace for '0xd15e59e4cca4aaf3ac57be8b71d35bb53549ec3ec1833da183d3fa5f4f609b83'
+Sender=0x1e59ce931B4CFea3fe4B875411e280e173cB7A9C
```

Comment:
Oh, this is clever!

wonder if you can call this `tx.origin` to match with the Solidity/Vyper environment variable exposing the origin account

Reply:
Done!